File: Java/dardo.java
import java.util.Locale;
import java.util.Scanner;

public class dardo {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		double dist1, dist2, dist3, maior;

		System.out.println("Digite as tres distancias:");
		dist1 = sc.nextDouble();
		dist2 = sc.nextDouble();
		dist3 = sc.nextDouble();

		if (dist1 > dist2 && dist1 > dist3) {
			maior = dist1;
		} else if (dist2 > dist3) {
			maior = dist2;
		} else {
			maior = dist3;
		}

		System.out.printf("MAIOR DISTANCIA = %.2f", maior);

		sc.close();
	}
}

File: C/dardo.c
#include <stdio.h>

int main()
{
    double dist1, dist2, dist3, maior;

    printf("Digite as tres distancias:\n");
    scanf("%lf", &dist1);
    scanf("%lf", &dist2);
    scanf("%lf", &dist3);

    if (dist1 > dist2 && dist1 > dist3){
        maior = dist1;
    } else if (dist2 > dist3){
        maior = dist2;
    } else {
        maior = dist3;
    }

    printf("MAIOR DISTANCIA = %.2lf", maior);
    return 0;
}
